bazel/bazel.go

```diff
@@ -237,10 +200,10 @@ func (b *bazel) Info() (map[string]string, error) {
 	defer close(doneCh)
 	go func() {
 		select {
-			case <- doneCh:
-				// Do nothing since we're done.
-			case <- time.After(8*time.Second):
-				log.Logf("Running `bazel info`... it's being a little slow")
+		case <-doneCh:
```

Comment:
Its interesting that this changed. What version of gofmt are you using?

Reply:
I'm just using vscode with the go version 1.13.15 which is in compliance with the one declared at https://github.com/bazelbuild/bazel-watcher/blob/master/go.mod#L15